### DISCUSSION TITLE
[Feature] Find & replace

### DISCUSSION THREAD

**Discussion by solvedDev** [bridge-core/bridge.#88] [Ideas]
**Summary**
Add find & replace functionality to bridge.

![image](https://user-images.githubusercontent.com/33347616/66503873-557b1800-eac8-11e9-905a-4ff224aec548.png)

**Comment by bschurman1** [comment]
![A7A915C1-730E-4D1A-B67A-71063A8BB518](https://user-images.githubusercontent.com/51575815/68078272-b680cd80-fda0-11e9-83ca-0336b12b60a6.jpeg)

**Comment by solvedDev** [comment]
Oops 🙃
